ui/imports/shared/views/AssetsDetailView.qml

```diff
@@ -54,12 +125,126 @@ Item {
         }
     }
 
-    ColumnLayout {
+    Loader {
+        id: graphDetailLoader
+        height: 290
+        anchors.left: parent.left
+        anchors.leftMargin: 24
+        anchors.right: parent.right
+        anchors.rightMargin: 24
         anchors.top: tokenDetailsHeader.bottom
         anchors.topMargin: 24
+        active: root.visible
+        sourceComponent: StatusChartPanel {
+            id: graphDetail
+            graphsModel: d.graphTabsModel
+            timeRangeModel: d.timeRangeTabsModel
+            onHeaderTabClicked: {
+                //TODO
+                //if time range tab
+                d.generateData();
+                //if graph bar
+                //switch graph
+            }
+            chart.chartType: 'line'
+            chart.chartData: {
+                return {
+                    labels: d.timeRange[graphDetail.timeRangeTabBarIndex][graphDetail.selectedTimeRange],
+                    datasets: [{
+                            label: 'Price',
+                            xAxisId: 'x-axis-1',
+                            yAxisId: 'y-axis-1',
+                            backgroundColor: (Theme.palette.name === "dark") ? 'rgba(136, 176, 255, 0.2)' : 'rgba(67, 96, 223, 0.2)',
+                            borderColor: (Theme.palette.name === "dark") ? 'rgba(136, 176, 255, 1)' : 'rgba(67, 96, 223, 1)',
```

Comment:
Should probably use colors coming from the Theme

Reply:
yes, it didn't work. for some reason it behaved strangely so I had to set it like this

Reply:
Ah, it expects HTML rgba() colors probably